nbclassic/templates/page.html

```diff
@@ -137,6 +137,26 @@
 </noscript>
 
 <div id="header" role="navigation" aria-label="{% trans %}Top Menu{% endtrans %}">
+  <div  id="newsId" style="display: none">
+    <div class="alert alert-info" role="alert">
+      <div style="display: flex">
+        <div>
+          <span class="label label-warning">NEWS</span>
+          If you plan to upgrade to Notebook 7, read <a href="https://nbclassic.readthedocs.io/en/latest/nbclassic_notebook.html" target="_top">https://nbclassic.readthedocs.io/en/latest/nbclassic_to_notebook7.html</a>
```

Comment:
Both the links in this line give me 404 errors. I'm guessing this is intentional and these docs aren't being pushed to `latest` until the release, but I wanted to note it just in case.

Reply:
Once the RDT website deployed (automatically done on PR merge), the 404 will be fixed.

Before releasing, we will double-check that the link is working fine.